tests/ieMode.e2e.runner.js

```diff
@@ -15,6 +16,5 @@ async function loadIEModeDriver () {
     return driver;
 };
 
-const driver_ie = await loadIEModeDriver();
-
-legacyRayCharles.runTests(driver_ie);
+gutenbergRo.runTests(await loadIEModeDriver(), ['jquery']);
```

Comment:
I wonder if we should run the basic test before the Gutenberg test? If basic is failing, then the more sophisticated ZIM is likely to fail too.